include/eve/arch/riscv/rvv_common_masks.hpp

```diff
@@ -0,0 +1,56 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/function/friends.hpp>
+
+namespace eve::detail
+{
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE logical<wide<T, N>>
+                rvv_true()
+{
+  static constexpr auto   lmul     = riscv_rvv_dyn_::getLMUL<T>(N::value);
+  static constexpr size_t size     = sizeof(T) * 8;
+  static constexpr size_t bit_size = lmul > 0 ? size / lmul : size * (-lmul);
```

Comment:
this seems to be copypasted a lot. Should we just have `logical<wide<T, N>>::platform_bit_size` or smth?

Also - really - this is changing depending on machine???

Reply:
> Should we just have logical<wide<T, N>>::platform_bit_size or smth?

This is not platform_bit_size of logical, it is the number that should be put to form right type. This number is equal to SEW/LMUL. Renamed it to `ratio`.

> Also - really - this is changing depending on machine?

Well, it depends on `VLEN` of your machine.

For example, you have `VLEN==128`. To operate with 8 int32 with one instruction, you will need to set LMUL( the number of consecutive vector registers that participate as one operand) to M2 (you need 2 registers). And mask for this you will need type `vbool16_t`(32/2=16).

If you have `VLEN==256`, you can operate 8 int32 with LMUL == M1, so you need to use `vbool32_t`(32/1=32).